core/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -468,7 +468,7 @@ public void testGreatest()
   {
     // Same types
     assertExpr("greatest(y, 0)", 2L);
-    assertExpr("greatest(34.0, z, 5.0, 767.0", 767.0);
+    assertExpr("greatest(34.0, z, 5.0, 767.0)", 767.0);
```

Comment:
👀  I'm surprised this didn't fail with a parse exception before

nice catch!

Reply:
it just writes parse error stuff to stderr but still ends up as a valid expression which is how the test passes I guess

```
line 1:28 missing ')' at '<EOF>'
line 1:28 missing ')' at '<EOF>'
```